dali/test/python/test_fw_iterators.py

```diff
@@ -149,6 +149,60 @@ def test_mxnet_iterator_last_batch_no_pad_last_batch():
     assert len(img_ids_list_set) == data_size
     assert len(set(mirrored_data)) != 1
 
+def test_mxnet_iterator_empty_array():
+    from nvidia.dali.plugin.mxnet import DALIGenericIterator as MXNetIterator
+
+    batch_size = 4
+    size = 100
+
+    class ExternalInputIterator(object):
+        def __iter__(self):
+            self.i = 0
+            self.n = size
+            return self
+
+        def __next__(self):
+            batch = []
+            bboxes = []
+            labels = []
+            for _ in range(batch_size):
+                batch.append(np.empty((224, 224, 3), dtype = np.uint8))
```

Comment:
I just tested that mxnet.nd.empty does'n work for np.longlong. But I don't have any good idea how to solve it.

Reply:
https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/ndarray/ndarray.py#L63 - maybe convert it to int64?

Reply:
What do you mean? It returns `int64` when `np.longlong` is used in source data. I've added the tests for this.

Reply:
```
>>> import mxnet as mx
>>> import numpy as np
>>> mx.nd.empty([1,2,3], dtype=np.longlong)
```
Crashes and I think DALI can return np.longlong.

Reply:
I've checked and after `pipe.run()` it already `int64`. Since MXNet iterator keeps this behavior maybe we can leave it like this and figure something out in separate task/PR?